Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticConsoleWriter.swift

```diff
@@ -42,7 +42,8 @@ public final class DiagnosticConsoleWriter: DiagnosticFormattingConsumer {
         highlight: Bool? = nil
     ) {
         outputStream = stream
-        formattingOptions = options
+        // Reenable `DefaultDiagnosticConsoleFormatter` as the default formatter when the index-out-of-bounds crash is fixed. https://github.com/apple/swift-docc/issues/722
+        formattingOptions = [.formatConsoleOutputForTools]
```

Comment:
Should we also pass `formattingOptions` into the call to `makeDiagnosticFormatter` just below? Right now it looks like we pass the original `options` value into `makeDiagnosticFormatter` which appears incorrect.

However, based on the conversation here we might end up just closing this PR out anyway...